dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/objectbundle/hooks/ProgramRuleVariableObjectBundleHook.java

```diff
@@ -60,6 +62,12 @@
         .put( ProgramRuleVariableSourceType.TEI_ATTRIBUTE, this::processTEA )
         .build();
 
+    private static final ImmutableCollection<ImportStrategy> UPDATE_STRATEGIES = ImmutableList.of(
```

Comment:
Change to `ImmutableSet` and `ImmutableSet.of`, as it is faster, and the order has no significance.

Reply:
true, I'll do it, ty